src/components/PacketTable/ReceiveTable/ReceiveTable.module.scss

```diff
@@ -0,0 +1,34 @@
+@use "@styles/styles";
+
+$custom-blue: #317ae7;
```

Comment:
It would be nice to establish a colour palette and set the same colours for everything.

Reply:
Yes. For now, this blue is only intended to be used in the receive table, but if it were to be used somewhere else, we'll make it global.